.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Context
+<!-- Describe the issues or requests addressed by this PR. Link to any relevant issues. -->
+
+## Changes
+<!-- Describe the changes proposed in this PR. -->
+
+## Testing
+<!-- Describe testing steps used to validate these changes, including automated and/or manual testing. -->
+
+- [ ] GH Actions workflows pass
```

Comment:
Hmm why do we need this? Are the merges not blocked by this explicit rn? If not, lmk and we can change that real quick which I'd prefer over this checkmark

Reply:
Included it only bc had seen similar pattern in one of our other repos, but def not needed for this one -- merges are blocked by failing workflows. Removed in [faddc18](https://github.com/AllenInstitute/biofile-finder/pull/430/commits/faddc1880d0d417c052c1d8c371a1e98354bb34d)